From 6df28433e5451b354c215ce65b850e6718b5a313 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Tue, 12 Jun 2018 17:22:30 +0100
Subject: [PATCH 2/2] Skip dh_apparmor when built with profile pkg.flatpak.app

---
 debian/control | 2 +-
 debian/rules   | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 57bf3202..578b0da4 100644
--- a/debian/control
+++ b/debian/control
@@ -7,7 +7,7 @@ Uploaders:
  Simon McVittie <smcv@debian.org>,
 Build-Depends:
  debhelper (>= 10),
- dh-apparmor [linux-any],
+ dh-apparmor [linux-any] <!pkg.flatpak.app>,
  dpkg-dev (>= 1.16.1),
  libcurl4-gnutls-dev,
  libjpeg-dev,
diff --git a/debian/rules b/debian/rules
index 1f2230e4..c45756e8 100755
--- a/debian/rules
+++ b/debian/rules
@@ -74,12 +74,14 @@ override_dh_auto_install:
 	cd $(DESTDIR)/usr/lib/quake3/ta/baseq3 && ln -s ../../../ioquake3/baseq3/*.so .
 	cd $(DESTDIR)/usr/lib/quake3/ta/missionpack && ln -s ../../../ioquake3/missionpack/*.so .
 
+ifeq ($(filter pkg.flatpak.app,$(DEB_BUILD_PROFILES)),)
 ifeq ($(DEB_HOST_ARCH_OS),linux)
 override_dh_install-arch:
 	dh_install -a
 	dh_apparmor -pioquake3 --profile-name=usr.lib.ioquake3.ioquake3
 	dh_apparmor -pioquake3-server --profile-name=usr.lib.ioquake3.ioq3ded
 endif
+endif
 
 override_dh_strip:
 	dh_strip --ddeb-migration='ioquake3-dbg (<< 1.36+u20160914+dfsg1-1~)'
-- 
2.17.1

